plugins/underflip/resorts/database/seeders/ResortsSeeder.php

```diff
@@ -53,6 +53,7 @@ public function run()
                 ),
                 '-'
             );
+            $resort->booking_affiliate_url = 'https://www.' . $resort->url_segment . '.com';
```

Comment:
```suggestion
            $resort->booking_affiliate_url = sprintf('https://localhost/%s', $resort->url_segment);
```

It's usually a good idea not to construct lifelike URLs for testing, in case a local environment or a CI environment starts sending out traffic on that URL.